packages/payments-plugin/src/mollie/mollie.handler.ts

```diff
@@ -1,17 +1,17 @@
-import createMollieClient, { RefundStatus } from '@mollie/api-client';
+import createMollieClient, { OrderEmbed, OrderStatus, PaymentStatus, RefundStatus } from '@mollie/api-client';
 import { LanguageCode } from '@vendure/common/lib/generated-types';
 import {
     CreatePaymentErrorResult,
     CreatePaymentResult,
     CreateRefundResult,
     Logger,
     PaymentMethodHandler,
-    PaymentMethodService,
     SettlePaymentResult,
 } from '@vendure/core';
-import { Permission } from '@vendure/core';
+import { PaymentState } from '@vendure/core/src';
```

Comment:
This is the cause of the build error. Should be imported from @vendure/core.

Reply:
Oops, so it IS caused by this PR :1st_place_medal: